### PR TITLE
Add top-level await

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: 
   * Node.js 15: https://github.com/nodejs/node/commit/54746bb763ebea0dc7e99d88ff4b379bcd680964
   * Node.js 14.8.0: https://github.com/nodejs/node/pull/34704
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: #7189

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Looking good to me!  Just a little nitpick to keep the scope of the PR small, can we separate the browser file changes into its own separate PR?

File: javascript/operators/await.json [hunk 1]
```diff
@@ -62,6 +62,75 @@
...
+        },
+        "top_level": {
+          "__compat": {
+            "description": "Use at module top level",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/await",
+            "spec_url": "https://tc39.es/proposal-top-level-await/#introduction",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "notes": "See <a href='https://crbug.com/9344'>bug 9344</a>"
```
Comment (ddbeck) [inline javascript/operators/await.json:74]: This link doesn't look right. I'm guessing a digit is missing. Could you find the correct link and fix it?

File: javascript/operators/await.json [hunk 2]
```diff
@@ -62,6 +62,75 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (ddbeck) [inline javascript/operators/await.json:129]: It looks like there's only one implementation right now. And I guess stage 3 proposals could conceivably change. Can we flip this one?

```suggestion
              "experimental": true,
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
A couple of minor issues in line comments, but otherwise good. Thank you, @saschanaz!

File: javascript/operators/await.json (hunk 1 above)
Comment (saschanaz) [reply javascript/operators/await.json:74]: Oops, https://bugs.chromium.org/p/v8/issues/detail?id=9344 is the bug I tried linking to. Should I just use the full URL?

Comment (saschanaz) [reply javascript/operators/await.json:74]: I got it, this should work
```suggestion
                "notes": "See <a href='https://crbug.com/v8/9344'>bug 9344</a>"
```

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you! 🎉

**Pull request merged**